### ISSUE TITLE
Advanced mode: WARNING - mismatch of the addEventListener property on type Window

### ISSUE THREAD

**Issue by 2beers** [google/closure-compiler#3372]
Compiling in advanced mode
```
		compilerOptions.setLanguageIn(LanguageMode.ECMASCRIPT_NEXT);
		compilerOptions.setLanguageOut(LanguageMode.ECMASCRIPT5);
```

I receive a lots of this warnings:
```
WARNING: externs/browser/w3c_dom1.js:123: WARNING - mismatch of the addEventListener property on type HTMLMediaElement and the type of the property it overrides from interface EventTarget
original: function(this:EventTarget, string, (EventListener|function(Event): *|null), (AddEventListenerOptions|boolean)=): undefined
override: function(this:Node, ?, ?, ?): ?
Node.prototype.addEventListener = function(type, listener, opt_options) {};
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

```
WARNING: externs/browser/w3c_dom1.js:851: WARNING - mismatch of the addEventListener property on type Window and the type of the property it overrides from interface EventTarget
original: function(this:EventTarget, string, (EventListener|function(Event): *|null), (AddEventListenerOptions|boolean)=): undefined
override: function(this:Window, ?, ?, ?): ?
Window.prototype.addEventListener = function(type, listener, opt_options) {};
```

```
WARNING: externs/browser/w3c_dom1.js:854: WARNING - mismatch of the removeEventListener property on type Window and the type of the property it overrides from interface EventTarget
original: function(this:EventTarget, string, (EventListener|function(Event): *|null), (EventListenerOptions|boolean)=): undefined
override: function(this:Window, ?, ?, ?): ?
Window.prototype.removeEventListener = function(type, listener, opt_options) {};
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

In my code I have something like `htmlElement.addEventListener("click",()=>{})`
I also call addEventListener from window object. Hope this helps.

I'm using v20190415.

**Comment by EatingW** [comment]
I think this is a result of 402f52a. @nreid260, can you confirm?

Please feel free to submit a pull request to fix the types
Node.prototype.addEventListener
Window.prototype.addEventListener
Window.prototype.removeEventListener

**Comment by EatingW** [comment]
Created internal issue http://b/132639077

**Comment by EatingW** [comment]
Actually Nick pointed out that 402f52a is not the issue, and that this current override should be valid. Please ignore the comment I made earlier. We'll be investigating this issue.
